src/dscanner/analysis/run.d

```diff
@@ -180,6 +181,32 @@ void generateReport(string[] fileNames, const StaticAnalysisConfig config,
 	writeln("}");
 }
 
+void generateSonarQubeGenericIssueDataReport(string[] fileNames, const StaticAnalysisConfig config,
+		ref StringCache cache, ref ModuleCache moduleCache)
+{
+	ulong lineOfCodeCount;
+	auto reporter = new SonarQubeGenericIssueDataReporter();
+
+	auto writeMessages = delegate void(string fileName, size_t line, size_t column, string message, bool isError){
+		reporter.addMessage(Message(fileName, line, column, "dscanner.syntax", message), isError);
+	};
+
+	foreach (fileName; fileNames)
+	{
+		auto code = readFile(fileName);
+		// Skip files that could not be read and continue with the rest
+		if (code.length == 0)
+			continue;
+		RollbackAllocator r;
+		const(Token)[] tokens;
+		const Module m = parseModule(fileName, code, &r, cache, tokens, writeMessages, &lineOfCodeCount, null, null);
```

Comment:
the `lineOfCodeCount` argument is not used anywhere and can be replaced with `null`

Reply:
fixed